Dockerfile.aarch64

```diff
@@ -53,7 +53,7 @@ RUN apt-get update && apt-get install -y \
 	python-mock \
 	python-pip \
 	python-websocket \
-	s3cmd=1.1.0* \
+	s3cmd=1.5.0* \
```

Comment:
Yeah, we should be able to axe this package entirely from all the architecture-specific Dockerfiles since we don't run releases in them (their primary purpose is generating artifacts). :+1: